src/screens/SendSheet.js

```diff
@@ -624,20 +625,22 @@ export default function SendSheet(props) {
       !selectedGasFee ||
       isEmpty(selectedGasFee?.gasFee)
     ) {
-      label = `Loading...`;
+      label = lang.t('button.confirm_exchange.loading');
       disabled = true;
     } else if (!isZeroAssetAmount && !isSufficientGas) {
       disabled = true;
-      label = `Insufficient ${nativeToken}`;
+      label = lang.t('button.confirm_exchange.insufficient_token', {
+        tokenName: nativeToken,
+      });
     } else if (!isValidGas) {
       disabled = true;
-      label = 'Invalid fee';
+      label = lang.t('button.confirm_exchange.invalid_fee');
```

Comment:
No change required just making a note that I confirm the change that `Invalid Fee` matches the casing pattern of these alerts (previously it was lowercase `fee`)